### PR TITLE
wip: add Spanish support

### PR DESCRIPTION
**Pull request by MichaelYuhe**
Add Spanish support for dashboard. Just using AI to translate those files and we may need further improvements by native Spanish speakers cc @dreyfus92

### CODE REVIEW

File: dashboard/locales/es-ES/common.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    "save": "Guardar",
+    "return": "Regresar",
+    "from": "Editar desde",
+    "all": "todos los entornos"
+  },
+  "empty-project": {
+    "desc": "El proyecto está vacío",
+    "click": "Haz clic",
+    "add": "Agregar Nuevo Servicio",
+    "start": "botón para comenzar!"
```
Comment (dreyfus92) [inline dashboard/locales/es-ES/common.json:12]:
```suggestion
    "start": "¡botón para comenzar!"
```

File: dashboard/locales/es-ES/deployment.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
+{
+    "deployed": "Desplegado",
+    "ago": "hace",
+    "status": {
+        "RUNNING": "EJECUTANDO",
+        "BUILDING": "CONSTRUYENDO",
+        "PENDING": "PENDIENTE",
+        "DEPLOYING": "DESPLIEGUE",
```
Comment (dreyfus92) [inline dashboard/locales/es-ES/deployment.json:8]:
```suggestion
        "DEPLOYING": "DESPLEGANDO",
```

**Review by dreyfus92 — COMMENTED** [review]
Awesome job @MichaelYuhe, I left some suggestions. I left it at `modals.json` I will review the rest later.

**Comment by MichaelYuhe** [comment]
> Awesome job @MichaelYuhe, I left some suggestions. I left it at `modals.json` I will review the rest later.

Thank you so much! I need to improve my Spanish :D

File: dashboard/locales/es-ES/service.json [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+            "toast": {
+                "success": "Directorio raíz cambiado",
+                "error": "Cambio de directorio raíz fallido"
+            }
+        },
+        "custom-build-command": {
+            "title": "Comando de Construcción Personalizado",
+            "desc": "Personaliza el comando de construcción para tu proyecto.",
+            "toast": {
+                "success": "Comando de construcción personalizado cambiado",
```
Comment (dreyfus92) [inline dashboard/locales/es-ES/service.json:100]:
```suggestion
                "success": "El Comando de construcción personalizado ha cambiado",
```

File: dashboard/locales/es-ES/service.json [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+                "error": "Cambio de comando de construcción personalizado fallido"
```
Comment (dreyfus92) [inline dashboard/locales/es-ES/service.json:101]:
```suggestion
                "error": "El Cambio de comando de construcción personalizado ha fallado"
```

File: dashboard/locales/es-ES/service.json [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+            }
+        },
+        "custom-start-command": {
+            "title": "Comando de Inicio Personalizado",
+            "desc": "Personaliza el comando de inicio para tu proyecto.",
+            "toast": {
+                "success": "Comando de inicio personalizado cambiado",
```
Comment (dreyfus92) [inline dashboard/locales/es-ES/service.json:108]:
```suggestion
                "success": "El Comando de inicio personalizado ha cambiado",
```

File: dashboard/locales/es-ES/service.json [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+                "error": "Cambio de comando de inicio personalizado fallido"
```
Comment (dreyfus92) [inline dashboard/locales/es-ES/service.json:109]:
```suggestion
                "error": "El Cambio de comando de inicio personalizado ha fallado"
```

File: dashboard/locales/es-ES/service.json [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+            }
+        },
+        "output-dir": {
+            "title": "Directorio de Salida Personalizado",
+            "desc": "El directorio que contiene la salida de construcción de tu proyecto. Si especificas un directorio, desplegaremos tu servicio como archivos estáticos con Nginx.",
+            "toast": {
+                "success": "Directorio de salida cambiado",
```
Comment (dreyfus92) [inline dashboard/locales/es-ES/service.json:116]:
```suggestion
                "success": "El Directorio de salida ha cambiado",
```

File: dashboard/locales/es-ES/service.json [hunk 8]
```diff
@@ -0,0 +1,148 @@
...
+                "error": "Cambio de directorio de salida fallido"
```
Comment (dreyfus92) [inline dashboard/locales/es-ES/service.json:117]:
```suggestion
                "error": "El Cambio de directorio de salida ha fallado"
```

File: dashboard/locales/es-ES/service.json [hunk 9]
```diff
@@ -0,0 +1,148 @@
...
+            }
+        },
+        "watch-path": {
+            "title": "Rutas de Vigilancia",
+            "desc": "Patrones al estilo Gitignore que se pueden usar para disparar nuevos despliegues basados en qué rutas de archivos han cambiado.",
+            "toast": {
+                "success": "Rutas de vigilancia cambiadas",
```
Comment (dreyfus92) [inline dashboard/locales/es-ES/service.json:124]:
```suggestion
                "success": "Las Rutas de vigilancia han sido cambiadas",
```

File: dashboard/locales/es-ES/service.json [hunk 10]
```diff
@@ -0,0 +1,148 @@
...
+                "error": "Cambio de rutas de vigilancia fallido"
```
Comment (dreyfus92) [inline dashboard/locales/es-ES/service.json:125]:
```suggestion
                "error": "El Cambio de rutas de vigilancia ha fallado"
```

File: dashboard/locales/es-ES/settings.json [hunk 11]
```diff
@@ -0,0 +1,50 @@
+{
+    "title": "Configuraciones",
```
Comment (dreyfus92) [inline dashboard/locales/es-ES/settings.json:2]:
```suggestion
    "title": "Configuración",
```

File: dashboard/locales/es-ES/toolbar.json [hunk 12]
```diff
@@ -0,0 +1,4 @@
+{
+    "add": "Agregar nuevo servicio",
+    "settings": "Configuraciones"
```
Comment (dreyfus92) [inline dashboard/locales/es-ES/toolbar.json:3]:
```suggestion
    "settings": "Configuración"
```

File: dashboard/locales/es-ES/header.json [hunk 13]
```diff
@@ -0,0 +1,13 @@
+{
+  "dashboard": "Tablero",
+  "upgrade": "Mejorar",
+  "docs": "Documentos",
+  "languages": "Idiomas",
+  "templates": "Plantillas",
+  "menu": {
+    "settings": "Configuraciones",
```
Comment (dreyfus92) [inline dashboard/locales/es-ES/header.json:8]:
```suggestion
    "settings": "Configuración",
```

**Review by dreyfus92 — COMMENTED** [review]
More small suggestions 😁

**Review by ScmTble — APPROVED** [review]

**Review by ScmTble — APPROVED** [review]

**Pull request merged**